File: text/java/read/text_read.java
// ----------------------------------------------------------------
/*
	read/text_read.java

					Sep/05/2016

*/
// ----------------------------------------------------------------
import	java.util.HashMap;

// ----------------------------------------------------------------
public class text_read
{
// ----------------------------------------------------------------
public static void main (String[] args)
{
	System.err.println ("*** 開始 ***");

	String	fname_in = args[0];

	HashMap <String, HashMap<String,String>> dict_aa
		= text_manipulate.text_read_proc (fname_in);

	text_manipulate.dict_display_proc (dict_aa);

	System.err.println ("*** 終了 ***");
}
// ----------------------------------------------------------------
}
// ----------------------------------------------------------------

File: text/scala/read/text_read.scala
// -----------------------------------------------------------------
//	text_read.scala
//
//						Sep/05/2016
// -----------------------------------------------------------------
import scala.io.Source
import scala.collection.mutable

// -----------------------------------------------------------------
object text_read
{
// -----------------------------------------------------------------
def main (args: Array[String])
{
	System.err.println ("*** 開始 ***")

	if (0 < args.length)
		{
		val file_in = args(0)

		var dict_aa = text_manipulate.text_read_proc (file_in)
		text_manipulate.dict_display_proc (dict_aa)
		}
	else
		{
		Console.err.println ("*** error ***")
		}

	System.err.println ("*** 終了 ***")
}

// -----------------------------------------------------------------
}
// -----------------------------------------------------------------
